lesson05/assignment/css/_components.content.scss

```diff
@@ -8,6 +10,485 @@ html {
+                    grid-row: 3;
+                }
+                @include responsive(desktop) {
+                    grid-column: 8 / 12;
+                    grid-row: 3;
+                }
+                @include responsive(desktopLarge) {
+                    grid-column: 7 / 14;
+                    grid-row: 4;
+                }
+               
+                // h4 article title 
+                &--title {
+                    margin: $xxs-rem 0;
+                }
+                // Paragraph
+                &--text { 
+                    font-family: $p-font;
+                    line-height: $m-rem;
+                }
+            // third article ends here
+            }
+       
+            // Sidebar
+            // Aside 
+                &__sidebar {
+           
+                width: 100%;
+                @include responsive(phone) {
+                    @include article-mobile-tablet;
+                }
+                @include responsive(tablet) {
+                    grid-column: 2 / 9;
+                    grid-row: 4;
+                }
+                @include responsive(tabletLarge) {
+                    grid-column: 2 / 9;
+                    grid-row: 4;
+                }
+                @include responsive(tabletXLarge) {
+                    grid-column: 2 / 5;
+                    grid-row: 2 / 4;
+                }
+                @include responsive(desktop) {
+                    grid-column: 2 / 5;
+                    grid-row: 2 / span 2;
+                }
+                @include responsive(desktopLarge) {
+                    grid-column: 2 / 6;
+                    grid-row: 2 / span 3;
+                }
+            // h3 sidebar title 
+             &--title {
+                margin: $m-rem 0;
+            }
+            &--img1 {
+                width: 50%;
+                object-fit: fill;
+                border-radius: 50%;
+
+            }
+            
+            &--text {
+                margin: $m-rem 0; 
+                font-family: $p-font;
+                line-height: $m-rem;
+                @include responsive(tabletXLarge) {
+                    &:last-child {
+                        display: none;
+                    }
+                }
+                @include responsive(desktop) {
+                    &:last-child {
+                        display: none;
+                    }
+                }
+            } 
+            &--img2 {
+                width: 96%;
+                object-fit: fill;
+                border-radius: 50%;
+                padding: 0 $m-rem;
+            }
+            &--img3 {
+                width: 96%;
+                object-fit: fill;
+                border-radius: 50%;
+                padding: 0 $m-rem;
+            }
+            &--img4 {
+                width: 96%;
+                object-fit: fill;
+                border-radius: 50%;
+                padding: 0 $m-rem;
+            }
```

Comment:
Since these are all the same set of styles, could they be compressed into a single block?
```
&--img2, &--img3, &--img4 {                
width: 96%;               
object-fit: fill;                
border-radius: 50%;                
padding: 0 $m-rem;            
}
```

Reply:
Grouping in just one line same set of styles makes sense. Thank you!!